integration_tests/mockApis/welcome.ts

```diff
@@ -157,7 +157,7 @@ export default {
     return stubFor({
       request: {
         method: 'POST',
-        urlPattern: `/welcome/transfers/${prisonNumber}/confirm`,
+        urlPath: `/welcome/transfers/${prisonNumber}/confirm`,
```

Comment:
It's not really that important checking to see if a hidden field is present on a page - that's more of an implementation detail. What would be good is to check that the create confirmation request is made with the arrival ID.

It would have been easy to hook into the existing request check code if it had been part of the post body... I'm not sure why it's a query parameter..